src/Kernel32.Tests.Shared/NTStatusExceptionFacts.cs

```diff
@@ -54,10 +58,11 @@ public void NTStatusException_Informational_Message()
     }
 
     [Fact]
+    [UseCulture("en-US")]
```

Comment:
We don't mean to test .NET Framework exception messages though, do we? If we're testing `FormatMessage` we can pass in the lcid and we don't need the attribute.